### ISSUE TITLE
Dependency Issues

### ISSUE THREAD

**Issue by kellystephen-10** [microsoft/Learn-LTI#110]
Hi,

After setting this up and configuring with an LMS, it returns the oops something is wrong error message when trying to launch.

Looking in the connect insights, this is a dependency issue

![image](https://user-images.githubusercontent.com/66949078/106020606-df833180-60bb-11eb-8e27-7b0c2a9d9f4b.png)

Have you seen this before?

Thanks,
SK

**Comment by nehalilani** [comment]
Hi @kellystephen-10 

Thank you for using the Microsoft Learn LTI tool. 
To get a better understanding of what might be causing the issue, could you please share a screenshot of the network tab from the browser's developer tools as well.

**Comment by kellystephen-10** [comment]
> Hi @kellystephen-10
> 
> Thank you for using the Microsoft Learn LTI tool.
> To get a better understanding of what might be causing the issue, could you please share a screenshot of the network tab from the browser's developer tools as well.

![image](https://user-images.githubusercontent.com/66949078/106024507-c2506200-60bf-11eb-93d4-15da1a0faebd.png)

**Comment by nehalilani** [comment]
The above screenshot indicates that the Users/me API call is not working as expected. I'd recommend you to follow the troubleshooting mentioned [here](https://github.com/microsoft/Learn-LTI/issues/55#issuecomment-694431630) 

>We'd request you to please share the above details with us in order for us to be able to help you in a better way. Since the details might contain some private information as well. Please feel free to reach out to us via email at learnlti@microsoft.com.

**Comment by leestott** [comment]
Hi Stephen so based on the information from your debugging

 
![image](https://user-images.githubusercontent.com/2511341/106456112-fee7d900-6484-11eb-82fa-5bcefa3b2acd.png)

We anticipate that the issue is due to the authorization headers being stripped when the tool requests members from the LMS. 

A way to confirm this is by using the ZTest Tool. https://ztest.cengage.info/ztest/ 

You can follow the steps listed here to set up the test tool.  https://tracker.moodle.org/browse/MDL-67072

Please let us know if this identifies the problem.

**Comment by leestott** [comment]
@kellystephen-10 thanks for the time. 

Glad we were able to resolve the root cause of the issue. 

Just as note for others
After they onboarded users to a Microsoft Azure Active Directory, we were able to get the NRPS working. However they were still not able to use the tool to validate users. 

The root cause has been identified: 
The email address with which the user was enrolled in the course within the LMS is not the same as the Azure Active Directory email. 

In your specific case the users addresses were the same however the casing CAPS/LOWER case being different. 
We were able to figure that to test this was the issue and this is the case. 

We have added an action item on our end to do the case matching in our code see https://github.com/microsoft/Learn-LTI/issues/114
